Kwxm/expose optimisers (PLT-1145) (#5156)
input-output-hk/plutus: plutus-core/changelog.d/20230220_085551_kenneth.mackenzie_expose_optimisers.md

### Added
- `optimise` options for the `pir`, `plc`, and `uplc` commands.
- A `compile` option for the `pir` command which allows a PIR file to be compiled to PLC or UPLC.

### Changed

> - Some of the `pir` commands have been extended to allow both `flat` and textual input.
> 

Reviewer: ```suggestion
- Some of the `pir` commands have been extended to allow also textual input (beside `flat` input).
```

Author: I will see if I can fix that.
